packages/wdio-shared-store-service/src/service.ts

```diff
@@ -1,7 +1,9 @@
 import type { JsonCompatible, JsonPrimitive, Services } from '@wdio/types'
 
-import { getValue, setValue, setPort } from './client.js'
+import { getValue, setValue, setPort, setResourcePool, getValueFromPool, addValueToPool } from './client.js'
 import type { SharedStoreServiceCapabilities } from './types.js'
+import type { JsonArray } from '@wdio/types'
```

Comment:
We already import types in line 1, let's merge these.

Reply:
oops, added now
will work on tests tomorrow